pwndbg/commands/context.py

```diff
@@ -131,6 +131,8 @@ def get_regs(*regs):
             continue
 
         value = pwndbg.regs[reg]
+        if not value and reg.lower() == "cpsr":
+            value = pwndbg.regs["xpsr"]
```

Comment:
Is this needed here? I don't like that we hardcode a specific register here; can this code be moved elsewhere?